samples/httpd/main.c

```diff
@@ -28,82 +28,82 @@ static void packet_timer(void *arg);
+    static ip4_addr_t ipaddr, netmask, gw;
 
 #if DEBUGGING
-	asm volatile ("jmp .");
-	debug_flags = LWIP_DBG_ON;
+    asm volatile ("jmp .");
+    debug_flags = LWIP_DBG_ON;
 #else
-	debug_flags = 0;
+    debug_flags = 0;
 #endif
 
 #if USE_DHCP
-	IP4_ADDR(&gw, 0,0,0,0);
-	IP4_ADDR(&ipaddr, 0,0,0,0);
-	IP4_ADDR(&netmask, 0,0,0,0);
+    IP4_ADDR(&gw, 0, 0, 0, 0);
+    IP4_ADDR(&ipaddr, 0, 0, 0, 0);
+    IP4_ADDR(&netmask, 0, 0, 0, 0);
 #else
-	IP4_ADDR(&gw, 10,0,1,1);
-	IP4_ADDR(&ipaddr, 10,0,1,7);
-	IP4_ADDR(&netmask, 255,255,255,0);
+    IP4_ADDR(&gw, 10, 0, 1, 1);
+    IP4_ADDR(&ipaddr, 10, 0, 1, 7);
+    IP4_ADDR(&netmask, 255, 255, 255, 0);
 #endif
 
-	/* Initialize the TCP/IP stack. Wait for completion. */
-	sys_sem_new(&init_complete, 0);
-	tcpip_init(tcpip_init_done, &init_complete);
-	sys_sem_wait(&init_complete);
-	sys_sem_free(&init_complete);
+    /* Initialize the TCP/IP stack. Wait for completion. */
+    sys_sem_new(&init_complete, 0);
+    tcpip_init(tcpip_init_done, &init_complete);
+    sys_sem_wait(&init_complete);
+    sys_sem_free(&init_complete);
 
-	XVideoSetMode(640, 480, 32, REFRESH_DEFAULT);
+    XVideoSetMode(640, 480, 32, REFRESH_DEFAULT);
 
-	pb_init();
-	pb_show_debug_screen();
+    pb_init();
+    pb_show_debug_screen();
 
-	g_pnetif = netif_add(&nforce_netif, &ipaddr, &netmask, &gw,
-	                     NULL, nforceif_init, ethernet_input);
-	if (!g_pnetif) {
-		debugPrint("netif_add failed\n");
-		return;
-	}
+    g_pnetif = netif_add(&nforce_netif, &ipaddr, &netmask, &gw,
+        NULL, nforceif_init, ethernet_input);
+    if (!g_pnetif) {
+        debugPrint("netif_add failed\n");
+        return 1;
+    }
 
-	netif_set_default(g_pnetif);
-	netif_set_up(g_pnetif);
+    netif_set_default(g_pnetif);
+    netif_set_up(g_pnetif);
 
 #if USE_DHCP
-	dhcp_start(g_pnetif);
+    dhcp_start(g_pnetif);
 #endif
 
-	packet_timer(NULL);
+    packet_timer(NULL);
 
 #if USE_DHCP
-	debugPrint("Waiting for DHCP...\n");
-	while (g_pnetif->dhcp->state != DHCP_STATE_BOUND)
-		NtYieldExecution();
-	debugPrint("DHCP bound!\n");
+    debugPrint("Waiting for DHCP...\n");
+    while (g_pnetif->dhcp->state != DHCP_STATE_BOUND)
+        NtYieldExecution();
+    debugPrint("DHCP bound!\n");
 #endif
 
-	debugPrint("\n");
-	debugPrint("IP address.. %s\n", ip4addr_ntoa(netif_ip4_addr(g_pnetif)));
-	debugPrint("Mask........ %s\n", ip4addr_ntoa(netif_ip4_netmask(g_pnetif)));
-	debugPrint("Gateway..... %s\n", ip4addr_ntoa(netif_ip4_gw(g_pnetif)));
-	debugPrint("\n");
+    debugPrint("\n");
+    debugPrint("IP address.. %s\n", ip4addr_ntoa(netif_ip4_addr(g_pnetif)));
+    debugPrint("Mask........ %s\n", ip4addr_ntoa(netif_ip4_netmask(g_pnetif)));
+    debugPrint("Gateway..... %s\n", ip4addr_ntoa(netif_ip4_gw(g_pnetif)));
+    debugPrint("\n");
 
-	http_server_netconn_init();
-	while (1) NtYieldExecution();
-	Pktdrv_Quit();
-	return;
+    http_server_netconn_init();
+    while (1) NtYieldExecution();
+    Pktdrv_Quit();
+    return 0;
```

Comment:
This file is now consistently spaces instead of tabs plus some spaces. I'll allow it.

Reply:
Generally a good idea, but I feel like we should have this in a separate commit.

Reply:
I reverted the spacing back to its original state, I think I hit auto format in VS in that file which caused that

Reply:
Seems to have worked out. Nice!